data/human/deep missions.txt

```diff
@@ -1989,8 +1989,11 @@ mission "Deep: Scientist Rescue 0"
 					accept
 				`	"Sorry, but this doesn't sound worth it to me."`
 					decline
-
+	on enter "Betelgeuse"
+		unmark "Betelgeuse"
+		dialog `Upon entering Betelgeuse, you fail to see any trace of the Star Queen. The pirates must have captured it and escaped to a nearby system.`
```

Comment:
You already know by this point that the Star Queen is somewhere else. The waypoint on Betelgeuse was because landing on Prime gives you (optional) reinforcements.